vllm/model_executor/layers/tuned_gemm.py

```diff
@@ -89,28 +112,7 @@ def mm(self, inp, weights):
                     })
                 ]).drop_duplicates()
                 self.tuned_df.to_csv(self.untune_path, index=False)
-
-            if ((n == 4 or n == 3 or n == 2 or n == 1) and k % 8 == 0
-                    and inp_view.dtype == torch.float16):
-                out = torch.empty(inp_view.shape[0],
-                                  weights.shape[0],
-                                  dtype=inp_view.dtype,
-                                  device='cuda')
-                _custom_C.wvSpltK(weights, inp_view, out, n, self.CuCount)
-            elif n == 1 and inp_view.dtype == torch.float16:
-                out = torch.empty(inp_view.shape[0],
-                                  weights.shape[0],
-                                  dtype=inp_view.dtype,
-                                  device='cuda')
-                if (k == 8192 and
-                    (m == 1280 or m == 7168)) or (k == 3584 and m == 8192):
-                    _custom_C.LLMM1(weights, inp_view, out, 8)
-                elif k <= 8192 and k % 8 == 0 and m % 4 == 0:
-                    _custom_C.LLMM1(weights, inp_view, out, 4)
```

Comment:
From the `wvSplitK` PR:
"I left in LLMM1 because it was restrictive in passed, and we haven't extensively checked if there are cases where it outperforms."
Are we sure that `wvSplitK` is strictly superior to LLMM1?

Reply:
No code path leads to it currently in out main

Reply:
I agree that the code path won't get hit here, so it's not necessary a live feature that's removed by this PR. But would be good to test that regardless.

Reply:
The other point is that if we're not going to touch `LLMM1`, should probably remove its C++ code too. But can be done in a follow-up PR.

Reply:
Oh and last point is I think the code path is hit if `k` is not a multiple of 8. We previously saw issues with that that were fixed [here ](https://github.com/ROCm/vllm/pull/28)?